_posts/2015-10-27-resetting-data-after-each-ui-test.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Resetting the application data after each test with Xcode 7 UI Testing
+permalink: /resetting-application-data-after-each-test-with-xcode7-ui-testing
+tags: ios, swift, xcode, testing, ui-testing
+categories: ios
+author: Tomáš Linhart
```

Comment:
@dirkaholic just proposed an update in PR #7 so you might need to rebase, update the new "_data/authors.yml" file and change that to `author_id:`.